src/views/ADempiere/Window/MultiTabWindow.vue

```diff
@@ -127,43 +126,23 @@ export default defineComponent({
     }
 
     const actionsManager = ref({
-      // overwrite logic or add actions
-      ...props.windowManager.actionsManager,
-
-      actionsList: ({ tableName, uuid }) => {
-        const actionsList = [
-          createNewRecord,
-          {
-            ...refreshRecords,
-            callBack: () => {
-              console.log('call getEntities')
-              root.$store.dispatch('dataManager/getEntities', {
-                parentUuid: props.parentUuid,
-                containerUuid: uuid,
-                tableName
-              })
-            }
-          },
-          deleteRecord,
-          sharedLink
-        ]
-
-        return actionsList
-      }
+      parentUuid: props.windowMetadata.uuid,
+      containerUuid: props.windowMetadata.currentTabUuid,
+      getActionList: () => [
```

Comment:
Where is implemented the actions list here? 
Where is implemented the actions click here?

Reply:
this returns the list of actions, they are imported as individual actions according to the implementation and added to the method